ssi-script.md

```diff
@@ -0,0 +1,122 @@
+
+_The Turing Way_ is a handbook for reproducible data science and collates together software tooling and best practices that makes this "too easy not to do"!
+I was particularly involved with incorporating Binder into the book.
+
+Through my contributions via _The Turing Way_, I was asked to officially join the Binder team where my role involves outreach as well as development.
+
+#### Slide 5
+
+From these experiences, I have developed some core values: the first of which is _reproducibility_.
+I strongly believe that the minimum bar for a project should be that the same data and analysis pipeline consistently produce the same results.
+
+#### Slide 6
+
+My next value is openness.
+Whether this is open science or open software, I believe building this kind of community with free exchanges of ideas can drive creative and scientific progress.
+
+#### Slide 7
+
+And lastly, I'm passionate about how Software Engineering can facilitate these values through tooling and best practices.
+
+#### Slide 8
+
+So what's this Binder project I'm so excited about?
+
+#### Slide 9
+
+mybinder.org is a free-to-use website that allows a code repository to be run in the cloud and easily shared via a single, clickable link.
+It achieves this by capturing a documented software environment from a public repository.
+
+#### Slide 10
+
+BinderHub is the computational infrastructure that underpins mybinder.org and collects together different tools, including JupyterHub and Kubernetes.
+
+#### Slide 11
+
+mybinder.org is a federated project, meaning that it is not hosted by a single team.
+The original Hub is hosted on Google Cloud in the US and a second Hub is hosted on OVH in Europe.
+
+#### Slide 12
+
+The Turing also has a plan to join the federation and host a Hub on Azure.
+Stay tuned!
+
+#### Slide 13
+
+How have I contributed so far?
+
+Through _The Turing Way_, I ran workshops that helped people make Binders out of their own projects.
+I have also helped more people than ever deploy their own BinderHubs.
+
+At this year's RSE Conference, I was invited to speak about collaborating with the Binder team on the RSE Worldwide panel, as well as running my BinderHub workshop.
+
+#### Slide 14
+
+What challenges do the Binder community face?
+
+The development team are very Python-centric which makes it tricky to help users with other languages or develop Binder into that space.
+
+The team is also quite small with few people knowing how to maintain the project, which can slow down development.
+
+#### Slide 15
+
+Why should someone be interested in joining the Binder community?
+
+They would have a voice in shaping the roadmap of the project and also help improve support for their language or domain.
+
+Not forgetting the opportunity to work with wonderful, supportive people!
+
+#### Slide 16
+
+For the SSI Fellowship, I have identified 4 conferences I would like to attend as a Binder advocate.
+The Fellowship should cover the cost of at least three.
```

Comment:
Not sure how difficult it will be to get accepted to these, but maybe it's worth adding something that you can do pro-actively e.g. host a hackathon in collaboration with the Turing Way community or something? It would be really cheap as you just need to pay food and maybe a train ticket for the odd helper, but doing something that just relies on you and not any conference committees to accept your proposals would make the application stronger I think.